content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -224,29 +137,28 @@ export KUBECONFIG=/etc/kubernetes/admin.conf
 ```
 
 Make a record of the `kubeadm join` command that `kubeadm init` outputs. You
-will need this in a moment.
+need this command to [join nodes to your cluster](#join-nodes).
 
 The token is used for mutual authentication between the master and the joining
-nodes.  The token included here is secret, keep it safe as anyone with this
+nodes.  The token included here is secret. Keep it safe, because anyone with this
 token can add authenticated nodes to your cluster. These tokens can be listed,
-created and deleted with the `kubeadm token` command. See the
-[reference guide](/docs/reference/setup-tools/kubeadm/kubeadm-token/) for more details.
+created, and deleted with the `kubeadm token` command. See the
+[kubeadm reference guide](/docs/reference/setup-tools/kubeadm/kubeadm-token/).
 
-### Installing a pod network {#pod-network}
+### Installing a pod network add-on {#pod-network}
 
 You **MUST** install a pod network add-on so that your pods can communicate with
 each other.
 
 **The network must be deployed before any applications.  Also, kube-dns, an
-internal helper service, will not start up before a network is installed. kubeadm only
-supports Container Network Interface (CNI) based networks (and does not support kubenet).**
+internal helper service, will not start up before a network is installed. kubeadm
+supports only Container Network Interface (CNI) based networks. It does not support kubenet.**
 
 Several projects provide Kubernetes pod networks using CNI, some of which also
-support [Network Policy](/docs/concepts/services-networking/networkpolicies/). See the [add-ons page](/docs/concepts/cluster-administration/addons/) for a complete list of available network add-ons. IPv6 support was added in [CNI v0.6.0](https://github.com/containernetworking/cni/releases/tag/v0.6.0). [CNI bridge](https://github.com/containernetworking/plugins/blob/master/plugins/main/bridge/README.md) and [local-ipam](https://github.com/containernetworking/plugins/blob/master/plugins/ipam/host-local/README.md) are the only supported IPv6 network plugins in 1.9.
-
+support [Network Policy](/docs/concepts/services-networking/networkpolicies/). See the [add-ons page](/docs/concepts/cluster-administration/addons/) for a complete list of available network add-ons. IPv6 support was added in [CNI v0.6.0](https://github.com/containernetworking/cni/releases/tag/v0.6.0). [CNI bridge](https://github.com/containernetworking/plugins/blob/master/plugins/main/bridge/README.md) and [local-ipam](https://github.com/containernetworking/plugins/blob/master/plugins/ipam/host-local/README.md) are the only supported IPv6 network plugins in Kubernetes version 1.9.
 
-**Note:** kubeadm sets up a more secure cluster by default and enforces use of [RBAC](#TODO).
-Please make sure that the network manifest of choice supports RBAC.
+**Note:** kubeadm sets up a more secure cluster by default and enforces use of [RBAC].
```

Comment:
More secure than what? I think RBAC needs link text. I don't think this needs to be a note. If you think it does, use the note shortcode.

How do I know if my network manifest supports RBAC?